data-prepper-plugins/kafka-connect-plugins/src/test/resources/sample-mysql-pipeline.yaml

```diff
@@ -0,0 +1,13 @@
+log-pipeline:
+  source:
+    mysql:
+      hostname: localhost
+      credentials:
+        plaintext:
+          username: debezium
+          password: dbz
+      tables:
+        - topic_prefix: prefix1
```

Comment:
Do we need this config ? topic_prefix: prefix1. ..Hope we can keep it optional and somehow use say connectorname.table name itself.

Reply:
We need this. It is mandatory by Debezium, and it has to be unique across other connectors. Moreover, it is required for OSI as well due to multi-tenant MSK